tests/helper/skyMapTestCase.py

```diff
@@ -64,13 +64,15 @@ def testRegistry(self):
     def testBasicAttributes(self):
         """Confirm that constructor attributes are available
         """
+        defaultSkyMap = self.getSkyMap()
         for tractOverlap in (0.0, 0.01, 0.1):  # degrees
             config = self.getConfig()
             config.tractOverlap = tractOverlap
             skyMap = self.getSkyMap(config)
             for tractInfo in skyMap:
                 self.assertAlmostEqual(tractInfo.getTractOverlap().asDegrees(), tractOverlap)
             self.assertEqual(len(skyMap), self._NumTracts)
+            self.assertNotEqual(skyMap.getSha1(), defaultSkyMap.getSha1())
```

Comment:
For at least some of these tests, can we now switch them to using the `__eq__` implementation?